mmdet/apis/test.py

```diff
@@ -17,12 +17,15 @@ def single_gpu_test(model,
                     data_loader,
                     show=False,
                     out_dir=None,
-                    show_score_thr=0.3):
+                    show_score_thr=0.3,
+                    time_monitor=None):
```

Comment:
😞

Reply:
agree, but I don't see here another solution. What about you?

Reply:
Seeing and thinking is generally not mine. But (quite random) guess is that `nn.Module.register_forward_hook` or `nn.Module.register_forward_pre_hook` might be suitable as well.

Reply:
thanks for the hint. done